ramls/edge-orders.raml

```diff
@@ -45,70 +58,99 @@ types:
         description: API Key
         type: string
   post:
-    description: Validate that the API Key provided can be used to place an order
+    description: Validate that the API Key provided can be used to place an order. The response body depends on the Vendor (GOBI or other)
     responses:
        200:
         description: Success
         body:
-          application/xml:
-           example: "<test>POST - OK</test>"
+         application/xml:
+          example: "<test>GET - OK</test>"
+         application/json:
+          example: |
+            {
+              "test" : "POST - OK"
+            }
        400:
         description: Bad Request
         body:
           application/xml:
-            type: orders
+            example: !include examples/error.xml
```

Comment:
Following my earlier comments...  This is the one place where it gets a little tricky since the error response might be generated by edge-orders itself instead of being a proxied response.  In this case I think what you have here is fine.  Maybe also include a text/plain as a fallback in the case where neither xml or json are acceptable.

Reply:
text/plain added in all error responses